TAF_EasyRest/TestFramework/PageComponents/AdministratorPanelComponents/AcceptedListPageComponent.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+namespace TestFramework.PageComponents.AdministratorPanelComponents
+{
+    public class AcceptedListPageComponent
+    {
+        IWebDriver driver { get; }
+        
+        private IReadOnlyCollection<IWebElement> _orders => driver.FindElements(By.XPath("//div[contains(@class, 'MuiTypography-root')]//div[contains(@class, ' MuiExpansionPanelSummary-root-450')]"));
+
+        public AcceptedListPageComponent(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        public AcceptedCardPageComponent ExpandAnyOrder()
```

Comment:
please rename to Expand**TheFirst**Waiter()

Reply:
renamed to "ExpandTheFirstOrder"